pupil_src/shared_modules/gaze_producer/controller/calibration_controller.py

```diff
@@ -51,7 +51,7 @@ def on_calibration_completed(status_and_result):
         )
         task.add_observer("on_completed", on_calibration_completed)
         task.add_observer("on_exception", tasklib.raise_exception)
-        self._task_manager.add_task(task)
+        self._task_manager.add_task(task, identifier=calibration.unique_id)
```

Comment:
```suggestion
        self._task_manager.add_task(task, identifier=f"{calibration.unique_id}-calibration")
```

Reply:
@papr is this blacked? Or written here on GitHub?

Reply:
Github

Reply:
Updated for black format